public/controllers/userController.js

```diff
@@ -6,10 +6,33 @@ angular.module('userController', ['userRegService'])
   _this = this;
   this.countryData;
   this.dropdCountries=[];
-
-
+  this.inputFields = {
+    'username' : '',
+    'fullname' : '',
+    'emailid'  : '',
+    'address'  : '',
+    'country'  : '',
+    'password' : '',
+    'retypepass' : '',
+   };
+
+  this.signupMessage = '';
   this.submitRegDetails = function(regData) {
-    if(validateRequired() && validatePassword() && validateEmailID()) {
+    _this.inputFields = {
+      'username' : '',
+      'fullname' : '',
+      'emailid'  : '',
+      'address'  : '',
+      'country'  : '',
+      'password' : '',
+      'retypepass' : '',
+    };
+
```

Comment:
@insomniac1426 consider refactoring the code so that you follow DRY principle. In this case you can create an inputFields object at say line 8 and assign it to this.inputEmpty fields as well as in line number 21. This reduces lines of code and incase you are making changes to the inputFields object you don't have to replicate that change at multiple places.